schemas/CreateAccountInput.json

```
@@ -0,0 +1,271 @@
+              "example": true,
+              "default": false
+            },
+            "repay_config": {
+              "type": "object",
+              "description": "Sensitive bank information will be stored as a secured token for payments in place of the raw account details.",
+              "required": [
+                "repay_payment_method",
+                "repay_check_type",
+                "repay_account_type",
+                "repay_transit_number",
+                "repay_account_number",
+                "repay_name_on_check"
+              ],
+              "properties": {
+                "repay_payment_method": {
+                  "type": "string",
+                  "enum": ["ACH"],
+                  "description": "The payment method to use for this account"
+                },
+                "repay_check_type": {
+                  "type": "string",
+                  "enum": ["PERSONAL", "BUSINESS"],
+                  "description": "Type of checking account: Personal or Business.",
+                  "example": "PERSONAL"
+                },
+                "repay_account_type": {
+                  "type": "string",
+                  "enum": ["SAVINGS", "CHECKING"],
+                  "description": "Type of account: Savings or Checking.",
+                  "example": "CHECKING"
+                },
+                "repay_transit_number": {
+                  "type": "integer",
+                  "description": "Transit number is a nine-digit code based on the U.S. Bank location where your account was opened.",
+                  "example": 999999992
+                },
+                "repay_account_number": {
+                  "type": "integer",
+                  "description": "Account number is an eight to ten digit number that identifies a specific account.",
+                  "example": 987654321
+                },
+                "repay_name_on_check": {
+                  "type": "string",
+                  "description": "Account holder's name as it appears on the account.",
+                  "example": "Lucille Bluth"
+                }
+              }
+            }
+          }
+        },
+        "cycle_type": {
+          "type": "object",
+          "properties": {
+            "first_cycle_interval": {
+              "type": "string",
+              "nullable": true,
+              "description": "Interval for a first cycle for this account.",
+              "example": "11 days"
+            }
+          }
+        },
+        "summary": {
+          "type": "object",
+          "properties": {
+            "credit_limit_cents": {
+              "type": "integer",
+              "description": "Total Amount (in cents) that this account can borrow.",
```

```suggestion
              "description": "Total amount (in cents) that this account can borrow.",
```

should `Amount` actually be capitalized?